### PR TITLE
document change for Release27Nov2020

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: docs/edge-logic/faq.md [hunk 1]
```diff
@@ -59,4 +59,4 @@ If you have one or more domains with ICP Beian and want them to be accelerated i
 
 ### How to support websocket?
 
-Use the directive [`enable_websocket`](</docs/edge-logic/supported-directives.md#enable_websocket>) in the location where websocket is needed. Make sure the client uses HTTP/1.1 (not HTTP/2) to connect. This directive also sets the read and send timeouts to 21s. It should not be used with the `origin_read_timeout` or `origin_send_timeout` directives in the same location.
+Use the directive [`enable_websocket`](</docs/edge-logic/supported-directives.md#enable_websocket>) in the location where websocket is needed. Make sure the client uses HTTP/1.1 (not HTTP/2) to connect. This directive also sets the read and send timeouts to 21s by default. They can be changed by the `origin_read_timeout` or `origin_send_timeout` directives.
```
Comment (burtonquantil) [inline docs/edge-logic/faq.md:62]: I suggest using "changed using" instead of "changed by".

File: docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ Denies access for the specified network or address. (Work in progress to make th
 **Default**: `-`<br/>
 **Context**: server, location
 
-This directive enables proxying the WebSocket protocol. The client must make sure not to use HTTP/2. The read and send timeouts are set to 21s. Do not use this directive with `origin_read_timeout` or `origin_send_timeout` in the same context.
+This directive enables proxying the WebSocket protocol. The client must make sure not to use HTTP/2. The default read and send timeouts are set to 21s and can be changed by the `origin_read_timeout` or `origin_send_timeout` directives.
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:111]: I believe "changed using" is better than "changed by". @marcfishelquantil

File: docs/edge-logic/supported-directives.md [hunk 3]
```diff
@@ -648,14 +647,16 @@ Assigns a value to the specified variable. No change to the public version. In p
...
 * We disallowed this directive in any "location" block to ensure the entire domain has the same slice size. This is to avoid potential problems when a request needs to be processed in multiple locations with different slice sizes.
 * CDN360 requires all cached slices to carry the same ETag value to ensure the content is consistent. When a slice fetched from the origin has a value that is different from the cached ones, any in-progress transfers to clients are terminated and all the cached slices are purged immediately. Please make sure the ETag value of each file on origin does not change unless the file's content has changed. This behavior can be disabled using `slice_ignore_etag on;`.
+* When slicing is enabled, the server automatically removes the `Accept-Encoding` header in the request to origin to disable compression. If this behavior is overridden, for example, by the `origin_set_header Accept-Encoding ...` directive, the client may receive corrupted response.
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:659]: I think the last sentence should end with "a corrupted response."

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I made a couple of minor suggestions.

File: docs/edge-logic/faq.md (hunk 1 above)
Comment (nibinqtl) [reply docs/edge-logic/faq.md]: updated.

File: docs/edge-logic/supported-directives.md (hunk 2 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: updated.

File: docs/edge-logic/supported-directives.md (hunk 3 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: updated.

**Review by burtonquantil — APPROVED** [review]
OK to merge.

**Review by burtonquantil — DISMISSED** [review]
OK to merge

**Comment by nibinqtl** [comment]
@burtonquantil @marcfishelquantil Release27Nov2020 has already been deployed. Can we push this PR to production?
Please approve it if you don't see any blocking issue.

**Review by burtonquantil — APPROVED** [review]
OK to merge. I keep approving but then there seem to be additional updates to dev which were reviewed and separately approved which cause the old review to become 'stale'.

**Pull request merged**